pootle/templates/editor/units/edit.html

```diff
@@ -152,19 +140,7 @@
                 </div>
                 {% endif %}
               </div>
-              <div id="js-unit-{{ unit.id }}"
-                   class="translate-original{% if unit.hasplural %} translate-plural{% endif %}">
-                {% for i, source, title in unit|pluralize_source %}
-                <div
-                  class="translation-text js-translation-text"
-                  lang="{{ source_language.code }}"
-                  dir="{{ source_language.direction }}"
-                  {% if title %} title="{{ title }}"{% endif %}
-                  data-string="{{ source|escapejs }}"
```

Comment:
Is there a reason why the `data-string` has been removed in the `UnitSource`?

FWIW until highlighting and copying is entirely done via components, this needs to remain, or we will be hitting some subtle bugs there (see https://github.com/translate/pootle/commit/a2ea4cf488e31647b7e9da67c766344e6962a5e6 for the detailed reasoning behind).